Module/Rule.Permission/Convert/Methods.ps1

```diff
@@ -577,20 +577,22 @@ function Split-MultiplePermissionRule
     $result = @()
     [System.Collections.ArrayList]$contentRanges = @()
     # Test for multiple paths at HKLMRoot
-    if ( $checkContent -match $regularExpression.hklmRootKeys )
+    if ($checkContent -match $regularExpression.hklmRootKeys)
     {
-        $hklmSecurityMatch = $checkContent | Select-String -Pattern $regularExpression.hklmSecurity
-        $hklmSoftwareMatch = $checkContent | Select-String -Pattern $regularExpression.hklmSoftware
-        $hklmSystemMatch   = $checkContent | Select-String -Pattern $regularExpression.hklmSystem
+        $hklmSecurityMatch  = $checkContent | Select-String -Pattern $regularExpression.hklmSecurity
+        $hklmSoftwareMatch  = $checkContent | Select-String -Pattern $regularExpression.hklmSoftware
+        $hklmSystemMatch    = $checkContent | Select-String -Pattern $regularExpression.hklmSystem
+        $lastPermissonMatch = $checkContent | Select-String -Pattern "\s-[\s\S]*?\s-" | Select-Object -Last 1
```

Comment:
This isn't consistent with the rest of the code.

Reply:
This regex should be added to the data file, then called from there, correct?  looks as if we've allowed this to slip thru at least one other time.

Reply:
Unfortunately it isn't consistent in this function, because there is RegEx that's used directly and some that's called from the data file.  I can change it to call from the data file, it just made the line longer.